crates/egui/src/input_state.rs

```diff
@@ -54,6 +54,22 @@ pub struct InputState {
     /// Position and size of the egui area.
     pub screen_rect: Rect,
 
+    /// Viewport inner position, only the drowable area
+    /// If is not working as expected is a winit bug!
+    pub inner_pos: Pos2,
+
+    /// Viewport outer position, drowable area + decorations
+    /// If is not working as expected is a winit bug!
+    pub outer_pos: Pos2,
+
+    /// Viewport inner size, only drowable area
+    /// If is not working as expected is a winit bug!
+    pub inner_size: Pos2,
```

Comment:
Instead of `inner_pos` and `inner_size`, use a `inner_rect: Rect` and `outer_rect: Rect`.

Reply:
The `Rect` i should create like:
```rust
Rect::from_min_size(viewport_pos, viewport_size);
```
Or
```rust
Rect::from_min_max(viewport_pos, viewport_size);
```

From what i see in the code base, i should use the first variant!